src/libs/VisualViewport/index.native.js

```diff
@@ -0,0 +1,13 @@
+
+/**
+ * Visual Viewport is not available on native, so return an empty function.
+ *
+ * @param {Function} onViewportResize
+ * @returns {Function}
+ */
+// eslint-disable-next-line no-unused-vars
```

Comment:
Can just remove the param if we're not using it

Reply:
I wasn't sure what was Kosher here, but sounds good, I'll do that.